sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -919,8 +926,9 @@ public Query apply(Function parent, Function inner, Context context) {
 
                 String parentName = functionLiteralPair.functionName();
 
-                Double from = null;
-                Double to = null;
+                GeoPoint geoPoint = new GeoPoint(lat, lon);
+                Double from = 0d;
+                Double to = GeoDistanceUtils.maxRadialDistanceMeters(geoPoint.lon(), geoPoint.lat());
```

Comment:
this caclulation seems to be overwritten later in some cases, please only do that if really required